benchmarks/ekobox/benchmark_evol_pdf.py

```diff
@@ -17,20 +17,20 @@
 def benchmark_evolve_single_member(
     tmp_path, cd, lhapdf_path, theory_card: TheoryCard, operator_card: OperatorCard
 ):
-    mu2grid = [20.0, 100.0, 10000.0]
+    mugrid = [(3.0, 4), (10.0, 5), (100.0, 5)]
```

Comment:
Even here I expected `np.sqrt(20)`

Reply:
Even here it should be irrelevant exactly which scale you're probing. It was a random number before, I used just another one, and avoided a function call.

Reply:
Ok looks fine